### PR TITLE
Merge rc140 into feat freeze account 2022.02.02

### PR DESCRIPTION
**Pull request by sstanculeanu**

### CODE REVIEW

File: data/esdt/esdt.pb.go [hunk 1]
```diff
@@ -28,10 +28,10 @@ var _ = math.Inf
 // proto package needs to be updated.
 const _ = proto.GoGoProtoPackageIsVersion3 // please upgrade the proto package
 
-// ESDigitalToken holds the data for a Elrond standard digital token transaction
+// ESDigitalToken holds the data for a Electronic standard digital token transaction
```
Comment (SebastianMarian) [inline data/esdt/esdt.pb.go:31]: electronic instead Electronic?

File: data/esdt/proto/esdt.proto [hunk 2]
```diff
@@ -8,10 +8,10 @@ option (gogoproto.stable_marshaler_all) = true;
 
 import "github.com/gogo/protobuf/gogoproto/gogo.proto";
 
-// ESDigitalToken holds the data for a Elrond standard digital token transaction
+// ESDigitalToken holds the data for a Electronic standard digital token transaction
```
Comment (SebastianMarian) [inline data/esdt/proto/esdt.proto:11]: electronic instead Electronic?

**Review by ssd04 — APPROVED** [review]

File: data/esdt/esdt.pb.go (hunk 1 above)
Comment (sstanculeanu) [reply data/esdt/esdt.pb.go]: pushed

File: data/esdt/proto/esdt.proto (hunk 2 above)
Comment (sstanculeanu) [reply data/esdt/proto/esdt.proto]: pushed

**Review by SebastianMarian — APPROVED** [review]

**Pull request merged**